<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(60, 120, 170);
QWidget {
font-size:14px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainVLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
background-color:rgb(70, 130, 180);
}


</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimumBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最小化&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QWidget {
background-color:rgb(70, 130, 180);
}

QPushButton {
	color:white;
	background-color: rgb(70, 130, 180);
	border:2px groovegray;
	padding:2px 4px;
}

QPushButton:hover {
/*   background-color: khaki;	*/
	background-color: rgb(135, 206, 250);
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
	background-color: rgb(173, 216, 230);
/*    background-color: #d0d67c;	*/
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QWidget {
background-color:rgb(70, 130, 180);
font: 14pt &quot;Consolas&quot;;
}

QPushButton {
	color:white;
	background-color: rgb(70, 130, 180);
	border:2px groovegray;
	padding:2px 4px;
}

QPushButton:hover {
/*   background-color: khaki;	*/
	background-color: rgb(150, 0, 0);
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
	background-color: rgb(220, 0, 0);
/*    background-color: #d0d67c;	*/
}

font-color:rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="logoWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 130, 180);</string>
     </property>
     <layout class="QHBoxLayout" name="logoHLayout" stretch="0">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 26pt &quot;楷体&quot;;
color:white;</string>
        </property>
        <property name="text">
         <string>泵车租赁管理系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="infoWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="infoHLayout">
      <item>
       <widget class="QWidget" name="picWidget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="loginWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
	background-color: rgb(173, 216, 230);
	border:2px groovegray;
	border-radius:10px;
	padding:2px 4px;
}

QPushButton:hover {
/*   background-color: khaki;	*/
	background-color: rgb(135, 206, 250);
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
	background-color: rgb(70, 130, 180);
/*    background-color: #d0d67c;	*/
}</string>
        </property>
        <widget class="QLabel" name="passwdLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;宋体&quot;;
</string>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
        <widget class="QLabel" name="unameLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="userNameLEdit">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="passwardLEdit">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="loginBtn">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>170</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
